#GY Calibrate
bool isGYCalibrate

# Throttle Values
int64 throttle